kko_okk/Views/HeaderViews/HeaderViewAssets.swift

```diff
@@ -85,4 +101,27 @@ class HeaderViewAssets{
     // 원본코드 출처 https://hururuek-chapchap.tistory.com/156
     
     
+    
+    var headerImageHeight:Double {
```

Comment:
```suggestion
    var headerImageHeight: Double {
```

: 뒤에는 스페이스를 하나 넣는게 Swift Style guide에 조금 더 맞습니다 ~

Reply:
넵!!